feathr-example/bureau_bureau-balance.ipynb

```diff
@@ -0,0 +1,1224 @@
+   "outputs": [],
+   "source": [
+    "# source for aggregated features of BUREAU\n",
+    "bureau_source_agg = HdfsSource(name=\"bureauSourceAgg\",\n",
+    "                          path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/bureau.csv\",\n",
+    "                          preprocessing=add_tran_date_column,\n",
+    "                          event_timestamp_column=\"TRAN_DATE\",\n",
+    "                          timestamp_format=\"yyyy-MM-dd HH:mm:ss\"\n",
+    "                          )\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# source for aggregated features\n",
+    "bureau_balance_source_core = HdfsSource(name=\"bureauBalanceSourceCore\",\n",
+    "                          path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/bureau_balance.csv\",\n",
+    "                          preprocessing=bureau_balance_preprocessing,\n",
+    "                          event_timestamp_column=\"TRAN_DATE\",\n",
+    "                          timestamp_format=\"yyyy-MM-dd HH:mm:ss\"\n",
+    "                          )\n",
+    "\n",
+    "f_MONTHS_BALANCE  = Feature(name=\"f_MONTHS_BALANCE\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING, \n",
+    "                        transform=\"MONTHS_BALANCE\")\n",
+    "\n",
+    "f_STATUS  = Feature(name=\"f_STATUS\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING,\n",
+    "                        transform=\"STATUS\")\n",
+    "\n",
+    "f_CREDIT_STATUS_EMA_AVG  = Feature(name=\"f_CREDIT_STATUS_EMA_AVG\",\n",
+    "                        key=key_SK_ID_BUREAU, \n",
+    "                        feature_type=STRING,\n",
+    "                        transform=\"CREDIT_STATUS_EMA_AVG\")\n",
+    "                        \n",
+    "features_bureau_balance_source_core=[\n",
+    "  f_MONTHS_BALANCE,\n",
+    "  f_STATUS,\n",
+    "  f_CREDIT_STATUS_EMA_AVG\n",
+    "  ]\n",
+    "\n",
+    "anchor_bureau_balance_source_core = FeatureAnchor(name=\"anchor_bureau_balance_source_core\",\n",
+    "                                source=bureau_balance_source_core,\n",
+    "                                features=features_bureau_balance_source_core)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# derived features BUREAU_BALANCE\n",
+    "\n",
+    "# source for aggregated features of BUREAU_BALANCE\n",
+    "# constant event_timestamp_column (1648655667 - \"epoch\")\n",
+    "bureau_balance_source_agg = HdfsSource(name=\"bureauBalanceSourceAgg\",\n",
+    "                          path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/bureau_balance.csv\",\n",
```

Comment:
azure storage account reference.

Reply:
Moved the resource prefix to a variable.